src/ovh-ui-angular/decisions.md

```diff
@@ -81,6 +89,22 @@ Only form events really used from the component external point-of-view should be
 
 On component that needs text inside of it but this text also needs to not contains html, like `oui-button`, the `text` attribute should be used.
 
+### Group together states that are exclusive in a single attribute
```

Comment:
Contradiction with
> Throw warning when concurrent modifiers are used at the same time

But "state" attribute vs "modifier" attribute... maybe it's right... but not clear!

Reply:
Removed since it was really in contradiction with the other.